sdk/common/caddyfile/caddyfile.go

```diff
@@ -0,0 +1,188 @@
+package caddyfile
```

Comment:
I don't think this belongs here, unless needed in the follow up PRs.

Reply:
thanks, fixed.